DICOM/src/Microsoft.Health.Dicom.Anonymizer.Core/Processors/SubstituteProcessor.cs

```diff
@@ -0,0 +1,73 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using Dicom;
+using EnsureThat;
+using Microsoft.Health.Dicom.Anonymizer.Core.Exceptions;
+using Microsoft.Health.Dicom.Anonymizer.Core.Model;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Health.Dicom.Anonymizer.Core.Processors
+{
+    public class SubstituteProcessor : IAnonymizerProcessor
+    {
+        private readonly string _replaceWith = "Anonymous";
```

Comment:
we can make this default value more unique like "REPLACED_BY_DICOM_ANONYMIZER"

Reply:
Fixed it.

Reply:
Kept it unchanged here, since "REPLACED_BY_DICOM_ANONYMIZER" will exceed length limitation for most items.